include/ninshiki_cpp/detector/color_detector.hpp

```diff
@@ -72,7 +72,7 @@ class ColorDetector : public Detector
   // Function for Contours
   void find(cv::Mat binary_mat);
 
-  void detection(cv::Mat image) override;
+  void detection(const cv::Mat & image);
```

Comment:
please don't remove `override` specifier, since this function indeed is overriding parent's function
https://github.com/ichiro-its/ninshiki_cpp/blob/1dbdf1f3c08011438604e834ab4529280266ecd4/include/ninshiki_cpp/detector/detector.hpp#L42